packages/typespec-ts/test/integration/modelPropertyAdditional.spec.ts

```diff
@@ -2,7 +2,7 @@ import { assert } from "chai";
 import AdditionalPropertiesClientFactory, {
   AdditionalPropertiesClient
 } from "./generated/models/propertyAdditional/src/index.js";
-describe("ClientRequestIdClient", () => {
+describe.only("ClientRequestIdClient", () => {
```

Comment:
```suggestion
describe("additional properties rest client", () => {
```
Please remember to remove .only when all changes is ready.